azurelinuxagent/common/utils/networkutil.py

```diff
@@ -185,13 +187,13 @@ def get_accept_tcp_rule(command, destination, firewalld_command="", wait=""):
     @staticmethod
     def get_iptables_accept_command(wait, command, destination, owner_uid):
         cmd = AddFirewallRules.__get_iptables_base_command()
-        cmd.extend(AddFirewallRules.__get_common_accept_command_params(wait, command, destination, owner_uid))
+        cmd.extend(AddFirewallRules.__get_common_accept_command_params(wait, command, destination, owner_uid, pos="2"))
```

Comment:
I have a question on how the position works for `iptables`. What if say the DNS rule and the DROP rule exist, so in this case if we add the ACCEPT command at `pos: 2`, will it be added at the right order? Or does it need the order of the rules to be in a specific order? 

Can there be a scenario where the rules are in a messed up order? Say the DROP rule is 1st and then the TCP rule is 3rd? What happens in that case? Will these code changes be able to handle that scenario?